ReactWindows/ReactNative.Shared/Bridge/ReactContext.cs

```diff
@@ -82,7 +82,7 @@ public T GetNativeModule<T>()
         /// Adds a lifecycle event listener to the context.
         /// </summary>
         /// <param name="listener">The listener.</param>
-        public virtual void AddLifecycleEventListener(ILifecycleEventListener listener)
+        public void AddLifecycleEventListener(ILifecycleEventListener listener)
```

Comment:
The only type that inherited from ReactContext was ThemedReactContext... Removing the virtual since there are no inheritors...